packages/usdk/lib/create.mjs

```diff
@@ -354,7 +354,6 @@ export const create = async (args, opts) => {
   const source = args.source;
   const features = typeof args.feature === 'string' ? JSON.parse(args.feature) : (args.feature || {});
   const yes = args.yes;
-  const force = !!args.force;
```

Comment:
If this flag is no longer needed, we can remove the option from the cli.

Reply:
Ah yes overlooked, removed